src/h5web/App.tsx

```diff
@@ -7,8 +7,10 @@ import BreadcrumbsBar from './BreadcrumbsBar';
 import Visualizer from './visualizer/Visualizer';
 import { getEntityAtPath } from './utils';
 import { ProviderContext } from './providers/context';
+import { assertAbsolutePath } from './guards';
 
 const DEFAULT_PATH = process.env.REACT_APP_DEFAULT_PATH || '/';
+assertAbsolutePath(DEFAULT_PATH);
```

Comment:
Almost everything now works with absolute paths, so we need to check for them from time to time.

I had a go at using TypeScript's new template literal types (e.g. `type AbsolutePath = `/${string}``, but inference was pretty poor. For instance, `const path: AbsolutePath = `/${entityName}` would not work. I had to use `as AbsolutePath` everywhere, which was really ugly. So I kept the `string` type throughout for paths, and used simple runtime assertions where relevant.